lib/rules/selector-type-no-unknown/__tests__/index.js

```diff
@@ -50,6 +50,16 @@ testRule(rule, {
   }, {
     code: "a:nth-child(n) {}",
   }, {
+    code: "input:not(*) {}",
```

Comment:
@hudochenkov see first note https://developer.mozilla.org/ru/docs/Web/CSS/:not :smile: it is possible